src/shared/types.ts

```diff
@@ -71,9 +76,10 @@ export const ScheduledSurfaces: ScheduledSurface[] = [
     guid: 'NEW_TAB_EN_GB',
     ianaTimezone: 'Europe/London',
     prospectTypes: [
-      ProspectType.GLOBAL,
-      ProspectType.ORGANIC_TIMESPENT,
-      ProspectType.APPROVED,
+      ProspectType.COUNTS,
```

Comment:
same way the acceptance criteria mentions we need to have `TitleUrlLearning` for all surfaces. is this same as `TITLE_URL_MODELED`?

Reply:
asking coz other surfaces have `TITLE_URL_MODELED` except this

Reply:
seems like  a DP requirement, following up these questions in this doc: https://getpocket.atlassian.net/wiki/spaces/PE/pages/2564587582/Prospecting+Sources#New-Tab-en-GB%3A--NEW_TAB_EN_GB

Looks good as per the docs.